website/content/en/docs/deployment/cognito-rds-s3/guide-terraform.md

```diff
@@ -102,6 +102,9 @@ pwd
     EOF
     ```
 
+### (Optional) Configure Culling for Notebooks
+Configure to enable notebook culling with the following [instructions]({{< ref "/docs/deployment/notebook-culling-configuration.md#" >}}). 
```

Comment:
wording is a bit off. maybe-

Enable culling for notebooks by following the instructions in configure culling for notebooks guide